tests/cleanup/remove_old_objects_azure.sh

```diff
@@ -0,0 +1,133 @@
+            rlRun -t -c 'az resource update --resource-type $resource_type --name $object --resource-group $AZURE_RESOURCE_GROUP --set "tags.$TAG_NAME=$now"' 0 "Add tag $TAG_NAME:$now to $resource_type: $object"
+        done
+    else
+        rlLogInfo "No $resource_type without the $TAG_NAME tag was found."
+    fi
+}
+
+rlJournalStart
+    rlPhaseStartSetup
+        if [ -z "$AZURE_SUBSCRIPTION_ID" ]; then
+            rlFail "AZURE_SUBSCRIPTION_ID is empty!"
+        else
+            rlLogInfo "AZURE_SUBSCRIPTION_ID is configured"
+        fi
+
+        if [ -z "$AZURE_TENANT" ]; then
+            rlFail "AZURE_TENANT is empty!"
+        else
+            rlLogInfo "AZURE_TENANT is configured"
+        fi
+
+        if [ -z "$AZURE_CLIENT_ID" ]; then
+            rlFail "AZURE_CLIENT_ID is empty!"
+        else
+            rlLogInfo "AZURE_CLIENT_ID is configured"
+        fi
+
+        if [ -z "$AZURE_SECRET" ]; then
+            rlFail "AZURE_SECRET is empty!"
+        else
+            rlLogInfo "AZURE_SECRET is configured"
+        fi
+
+        export AZURE_RESOURCE_GROUP="${AZURE_RESOURCE_GROUP:-composer}"
+        rlLogInfo "AZURE_RESOURCE_GROUP=$AZURE_RESOURCE_GROUP"
+
+        export AZURE_STORAGE_ACCOUNT="${AZURE_STORAGE_ACCOUNT:-composerredhat}"
+        rlLogInfo "AZURE_STORAGE_ACCOUNT=$AZURE_STORAGE_ACCOUNT"
+
+        export AZURE_STORAGE_CONTAINER="${AZURE_STORAGE_CONTAINER:-composerredhat}"
+        rlLogInfo "AZURE_STORAGE_CONTAINER=$AZURE_STORAGE_CONTAINER"
+
+        # VMs older than HOURS_LIMIT will be deleted
+        HOURS_LIMIT="${HOURS_LIMIT:-24}"
+        export TIMESTAMP=`date -u -d "$HOURS_LIMIT hours ago" '+%FT%T'`
+
+        rlLogInfo "HOURS_LIMIT=$HOURS_LIMIT"
+        rlLogInfo "TIMESTAMP=$TIMESTAMP"
+
+        # It's not easily possible to get creation date/time of Azure objects.
+        # Use a tag to record when the object was seen for the first time
+        # and remove objects based on the value of the tag. The value is UTC
+        # date/time, format: 2019-01-29T15:16:40
+        TAG_NAME="first_seen"
+
+        # Use Microsoft repository to install azure-cli
+        rlRun -t -c "rpm --import https://packages.microsoft.com/keys/microsoft.asc"
+        cat > /etc/yum.repos.d/azure-cli.repo << __EOF__
+[azure-cli]
+name=Azure CLI
+baseurl=https://packages.microsoft.com/yumrepos/azure-cli
+enabled=1
+gpgcheck=1
+gpgkey=https://packages.microsoft.com/keys/microsoft.asc
+__EOF__
+        rlRun -c -t "dnf install -y azure-cli"
+
+        # sign in
+        rlRun -c -t 'az login --service-principal --username "$AZURE_CLIENT_ID" --password "$AZURE_SECRET" --tenant "$AZURE_TENANT"'
+    rlPhaseEnd
+
+    # A list of Azure resources we want to clean
+    resource_types="
+Microsoft.Compute/virtualMachines
+Microsoft.Network/networkInterfaces
+Microsoft.Network/publicIPAddresses
+Microsoft.Network/networkSecurityGroups
+Microsoft.Compute/disks
+Microsoft.Compute/images
+Microsoft.Storage/storageAccounts
```

Comment:
Don't remove storage accounts. This already caused existing Azure tests to fail!

Reply:
A new storage account is created automatically when a new VM is created by the testing script test_build_and_deploy_azure.sh, so here are some possible solutions:
1) modify test_build_and_deploy_azure.sh to use existing storage account (composerredhat) instead of creating new one for each VM. I prefer this option.
2) do not delete storage accounts at all
3) delete storage accounts except of composerredhat

Reply:
+1 for 1. and 2.